src/implementations/vanilla/components/gallery/gallery.js

```diff
@@ -480,8 +483,19 @@ export class Gallery {
       mediaIframe.setAttribute('src', iframeUrl);
       mediaIframe.setAttribute('frameBorder', '0');
 
+      // Update iframe title
+      let videoTitle = '';
+      const title = queryOne(this.titleSelector, selectedItem);
+      if (title) {
+        videoTitle = title.innerHTML;
+      }
       if (this.videoPlayerLabel) {
-        mediaIframe.setAttribute('title', this.videoPlayerLabel);
+        videoTitle = videoTitle
```

Comment:
but also here...i mean, but why would we have to do something like this..? Retrieving two strings and combining them? It is also something that happens without the user being aware of that, why don't we simply expect the final label we construct from ONE single parameter, instead? It's a crazy overhead for no reason, in my opinion, the user will still have to pass two params and not knowing that they will be magically combined together, isn't it just better to expect from this videoPlayerLabel whatever the string to be added as the title of the iframe?

Reply:
I'm not sure that the editor would understand that they are supposed to use a label like "[video title] - Video player" (that's what is requested at accessibility level...)
What we could do is concatenate the string before sending it to the js, so we only check for one data-* thingy

Reply:
yes, that's already better, but my point of view is that user will not understand anyhow what to do with these things, the more they have the less they will understand..